doc/source/myrocks/variables.rst

```diff
@@ -1977,3 +2121,610 @@ which can be useful for bulk loading.
+in MyRocks system tables.
+
+.. variable:: rocksdb_memtable_total
+
+.. variable:: rocksdb_memtable_unflushed
+
+.. variable:: rocksdb_queries_point
+
+.. variable:: rocksdb_queries_range
+
+.. variable:: rocksdb_covered_secondary_key_lookups
+
+.. variable:: rocksdb_block_cache_add
+
+This variable shows the number of blocks added to block cache.
+
+.. variable:: rocksdb_block_cache_add
+
+This variable shows the number of blocks added to block cache.
+
+.. variable:: rocksdb_block_cache_add_failures
+
+This variable shows the number of failures when adding blocks to block cache.
+
+.. variable:: rocksdb_block_cache_bytes_read
+
+This variable shows the number of bytes read from cache.
+
+.. variable:: rocksdb_block_cache_bytes_write
+
+This variable shows the number of bytes written into cache.
+
+.. variable:: rocksdb_block_cache_data_add
+
+This variable shows the number of data blocks added to block cache.
+
+.. variable:: rocksdb_block_cache_data_bytes_insert
+
+This variable shows the number of bytes of data blocks inserted into cache.
+
+.. variable:: rocksdb_block_cache_data_hit
+
+This variable shows the number of cache hits when accessing the
+data block from the block cache.
+
+.. variable:: rocksdb_block_cache_data_miss
+
+This variable shows the number of cache misses when accessing the
+data block from the block cache.
+
+.. variable:: rocksdb_block_cache_filter_add
+
+This variable shows the number of filter blocks added to block cache.
+
+.. variable:: rocksdb_block_cache_filter_bytes_evict
+
+This variable shows the number of bytes of bloom filter blocks
+removed from cache.
+
+.. variable:: rocksdb_block_cache_filter_bytes_insert
+
+This variable shows the number of bytes of bloom filter blocks
+inserted into cache.
+
+.. variable:: rocksdb_block_cache_filter_hit
+
+This variable shows the number of times cache hit when accessing filter block
+from block cache.
+
+.. variable:: rocksdb_block_cache_filter_miss
+
+This variable shows the number of times cache miss when accessing filter
+block from block cache.
+
+.. variable:: rocksdb_block_cache_hit
+
+This variable shows the number of blocks added to block cache.
+
+.. variable:: rocksdb_block_cache_index_add
+
+This variable shows the number of index blocks added to block cache.
+
+.. variable:: rocksdb_block_cache_index_bytes_evict
+
+This variable shows the number of bytes of index block erased from cache.
+
+.. variable:: rocksdb_block_cache_index_bytes_insert
+
+This variable shows the number of bytes of index blocks inserted into cache.
+
+.. variable:: rocksdb_block_cache_index_hit
+
+This variable shows the number of index blocks added to block cache.
+
+.. variable:: rocksdb_block_cache_index_miss
+
+This variable shows the number of times cache hit when accessing index
+block from block cache.
+
+.. variable:: rocksdb_block_cache_miss
+
+This variable shows the total number of block cache misses.
+
+.. variable:: rocksdb_block_cachecompressed_hit
```

Comment:
@georgelorchpercona how come a missing underscore here? Possible to fix? Or is "cachecompressed" a thing, like "GetUpdatesSince" below?

Reply:
That's how it is defined in the code.

Reply:
Yes, my question is whether it's possible to fix it in the code?

Reply:
This seems to be in the MyRocks translation from RocksDB, not in RocksDB so yes, it looks possible to fix by just changing the name in the translation macro.

Reply:
https://jira.percona.com/browse/MYR-212

Reply:
https://github.com/percona/percona-server/pull/2003